decidim-comments/lib/decidim/comments/test/shared_examples/comment_event.rb

```diff
@@ -42,4 +42,21 @@
       expect(subject.resource_text).to eq comment.formatted_body
     end
   end
+
+  context "when is deleted" do
+    let(:comment) { create :comment, :deleted }
+
+    it "outputs the comment body" do
+      expect(subject.resource_text).to include("Comment deleted on")
+    end
```

Comment:
How about making this a bit more explicit? I was thinking on something like this:
```suggestion
    it "does not output the comment body" do
      expect(subject.resource_text).to include("Comment deleted on")
      expect(subject.resource_text).not_to include(comment.body)
    end
```

We should apply it to also the "when is moderated" block